### ISSUE TITLE
weblate.trans.tests.test_vcs fails with git-2.15.0

### ISSUE THREAD

**Issue by stanislav-brabec** [WeblateOrg/weblate#1693]
Changes between Git 2.13.1 and Git 2.15.0 trigger failures of weblate.trans.tests.test_vcs.

### Steps to reproduce
1. Build weblate with git-2.15.0

### Actual behaviour
```
======================================================================
FAIL: test_status (weblate.trans.tests.test_vcs.VCSGerritTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/abuild/rpmbuild/BUILD/Weblate-2.17.1/weblate/trans/tests/test_vcs.py", line 227, in test_status
    "Your branch is up-to-date with 'origin/master'.", status
AssertionError: u"Your branch is up-to-date with 'origin/master'." not found in u"On branch master\nYour branch is up to date with 'origin/master'.\n\nnothing to commit, working tree clean\n"

======================================================================
FAIL: test_status (weblate.trans.tests.test_vcs.VCSGitTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/abuild/rpmbuild/BUILD/Weblate-2.17.1/weblate/trans/tests/test_vcs.py", line 227, in test_status
    "Your branch is up-to-date with 'origin/master'.", status
AssertionError: u"Your branch is up-to-date with 'origin/master'." not found in u"On branch master\nYour branch is up to date with 'origin/master'.\n\nnothing to commit, working tree clean\n"

======================================================================
FAIL: test_status (weblate.trans.tests.test_vcs.VCSGithubTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/abuild/rpmbuild/BUILD/Weblate-2.17.1/weblate/trans/tests/test_vcs.py", line 227, in test_status
    "Your branch is up-to-date with 'origin/master'.", status
AssertionError: u"Your branch is up-to-date with 'origin/master'." not found in u"On branch master\nYour branch is up to date with 'origin/master'.\n\nnothing to commit, working tree clean\n"

----------------------------------------------------------------------
```
### Expected behaviour
Success.
```patch
Index: Weblate-2.17.1/weblate/trans/tests/test_vcs.py
===================================================================
--- Weblate-2.17.1.orig/weblate/trans/tests/test_vcs.py
+++ Weblate-2.17.1/weblate/trans/tests/test_vcs.py
@@ -224,7 +224,7 @@ class VCSGitTest(TestCase, RepoTestMixin
     def test_status(self):
         status = self.repo.status()
         self.assertIn(
-            "Your branch is up-to-date with 'origin/master'.", status
+            "Your branch is up to date with 'origin/master'.", status
         )
 
     def test_needs_commit(self):
```
But you probably need two checks or regex match to cover both git versions.
### Server configuration
Not applicable.

**Comment by nijel** [comment]
Already fixed in eb9b9cb6c21110f3f6e5f2bcb01ceb0ad802c2b4.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
